compiler/src/dotty/tools/dotc/quoted/reflect/ReflectionCompilerInterface.scala

```diff
@@ -1333,9 +1338,7 @@ class ReflectionCompilerInterface(val rootContext: Context) extends CompilerInte
   }
 
   def AppliedType_tycon(self: AppliedType)(using Context): Type = self.tycon
-  def AppliedType_args(self: AppliedType)(using Context): List[TypeOrBounds] = self.args
-
-  def AppliedType_apply(tycon: Type, args: List[TypeOrBounds])(using Context): AppliedType = Types.AppliedType(tycon, args)
```

Comment:
Maybe keep it for one release but deprecated in favor of appliedTo?

Reply:
We don't have enough releases anymore to do the conservative approach.